docs/organizations/verify-user-permissions.mdx

```diff
@@ -11,21 +11,11 @@ description: A collection of utility functions and components in order to allow
 
 Clerk provides two kinds of helpers for authorization:
```

Comment:
Out of scope of this PR, but we should discuss whether we want to speak about our company in third or first person. This doc uses third, but [others use first person](https://clerk.com/docs/quickstarts/setup-clerk#create-a-clerk-application). Either is fine IMO, as long as we're consistent

![image](https://github.com/clerk/clerk-docs/assets/5156915/e413b0d5-db97-4620-8297-89dfdc7c4814)

cc @kylemac @jescalan for thoughts

Reply:
We have a style guide @S3Prototype that @alexisintech has been steaduly updating old docs to adhere to. See https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md

Reply:
@royanger Thanks!